src/core/core.controller.js

```diff
@@ -923,7 +924,7 @@ module.exports = function(Chart) {
 			}
 
 			// Built in hover styling
-			if (me.active.length && hoverOptions.mode) {
+			if (me.active.length && hoverOptions.mode && helpers.valueOrDefault(hoverOptions.enabled, true)) {
```

Comment:
I think we could make a variable `hoverEnabled = valueOrDefault(me.options.hover.enabled, true)` since we would use it both places that were changed